lib.ftd

```diff
@@ -2529,36 +2528,32 @@ if: { ftd.device != "mobile" }
 src: $lezgoo.image
 
 -- ftd.column:
-padding.px if { ftd.device == "mobile" }: 48
+padding.px if { ftd.device == "mobile" }: 40
 
 -- ftd.row:
 margin-bottom.px: 24
 wrap: true
 
 -- ftd.text: $lezgoo.title
 role: $inherited.types.heading-small
-color: $inherited.colors.cta-primary.base
-
--- ftd.text: 
-role: $inherited.types.heading-small
-color: $inherited.colors.text
-padding-left.px: 10
-
-$lezgoo.body1
-
--- ftd.text: 
-role: $inherited.types.heading-small
 color: $inherited.colors.text
+max-width.fixed.percent if { ftd.device != "mobile" }: 80
 
-$lezgoo.body
+;;-- ftd.text: 
+;;role: $inherited.types.heading-small
+;;color: $inherited.colors.text
+;;
+;;$lezgoo.body
 
 -- end: ftd.row
 
 -- ftd.text: $lezgoo.cta-label
 background.solid: $inherited.colors.cta-primary.base
 padding-horizontal.px: 41
 padding-vertical.px: 11
-border-radius.px: 50
+border-radius.px if { ftd.device != "mobile" }: 50
+border-radius.px if { ftd.device == "mobile" }: 34
+role: $inherited.types.button-large
 color: $inherited.colors.cta-primary.text
 link if { lezgoo.cta-url != NULL }: $lezgoo.cta-url
```

Comment:
```suggestion
link: $lezgoo.cta-url
```
@SuryanshShahi remove condition, does it make sense to show button without link?

Reply:
done!